data/korath/korath weapons.txt

```diff
@@ -53,6 +54,64 @@ effect "grab-strike impact"
 	"velocity scale" -.1
 
 
+outfit "Korath Shunt-Strike"
+	category "Turrets"
+	cost 821000
+	thumbnail "outfit/shunt-strike"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
+	"energy capacity" 260
+	"turret mounts" -1
+	"required crew" 1
+	"asteroid scan power" 16
+	weapon
+		sprite "projectile/shunt-strike"
+			"frame rate" 24
+			"random start frame"
+		"hardpoint sprite" "hardpoint/shunt-strike"
+		"hardpoint offset" 7.
+		sound "shunt-strike"
+		"inaccuracy" 1
+		"turret turn" 2.1
+		"velocity" 10
+		"lifetime" 81
+		"reload" 30
+		"hit effect" "shunt-strike hit"
+		"die effect" "shunt-strike sparkle"
+		"firing energy" 100
+		"firing heat" 150
+		"relative firing shields" 0.001
+		"acceleration" 1
+		"drag" .1
+		"turn" .2
+		"homing" 4
+		"tracking" .9
+		"shield damage" 48
+		"corrosion damage" 3
+		"minable damage" 200
+		"piercing" 0.1
+		"blast radius" 30
+		"safe"
+		"missile strength" 80
+	description "This weapon fires homing balls of microbots that pass through a target's shields and attempt to convert its hull into raw materials or useful items such as heat shunts. They inevitably fail, as Exile microbots cannot sustain themselves without a Systems Core, but not before wreaking havoc on the ship."
```

Comment:
So take this with a grain of salt. My first thought would have been that the issue would be them running out of commands in the buffer/orders to process and so stuttering to a halt. Essentially expending their usefulness. 
This kind of suggests they either run out of power or break down after a short time, and I'd have thought they'd need to have a decent duration of power to do their job normally - though if they break down and are just normally picked up and recycled by other microbots then that's a different matter and this is fine.

In any sense, if the idea is that they just break down over time then all good, but if it was something else I think it might be worth potentially adding a little more clarity here.
```suggestion
	description "This weapon fires homing balls of microbots that pass through a target's shields and attempt to convert its hull into raw materials or useful items such as heat shunts. They inevitably fail, as Exile microbots cannot perform sustained commands themselves without a Systems Core, but not before wreaking havoc on the ship."
```

Reply:
Hm.  It's an open question, but I was imagining multiple factors.

One being like you suggest, that they lack the computing power to be fully autonomous, relying on communication with the Systems Core when the unexpected comes up.

The other, more significant issue in my mind, being that they may not have internal energy generation, running entirely on capacitors.  Without a Systems Core to either return to or beam power to them wirelessly, they'd simply starve for energy.

Reply:
Ah wireless power - I would think it more likely that they just recharge (if needed) off the nearest power conduit.

Reply:
Assuming they're compatible - not much room for adapters in a 1mm - 1um package!  I imagine, for simplicity and safety, they'd be keyed only to accept particular voltages or frequencies, in a particular manner.

Reply:
They could potentially recharge wirelessly in a more local sense (rather than beamed from the system core) from the magnetic field around a power conduit. Inductive charging style - no adaptors required, and no beaming from the system core.

Reply:
I think you miss my point
The lack of rechargeability outside of specific methods would be a *deliberate safety feature* to prevent microbots from doing too much on their own.  No one wants another rampant swarm.

Reply:
Ah yes, I see - though that could be a non-issue if the limiting factor was the number of potential task/commands they could store up and proceed to carry out. A rampant swarm of bots that just sits around and charges themselves up because they've run out of other tasks to do are pretty safe.
Likewise, if doing microbot stuff causes them to wear down and break - and the systems core is needed to print off new ones, that's a pretty safe swarm too.

Reply:
The problem I have with commands is that it's a *software* limitation.  If a few microbots out of the billions in operation suffer the wrong glitch... it becomes a problem.

A hardware lockout makes this more difficult.  Having them just wear out would *work*, but I don't think it's much fun to imagine.  I much prefer needing to recharge.